pyaedt/modules/Boundary.py

```diff
@@ -327,6 +327,32 @@ def delete(self):
         return True
 
 
+class NativeComponentPCB(NativeComponentObject, object):
+    """Manages Native Component PCB data and execution.
+
+    Parameters
+    ----------
+    app : object
+        An AEDT application from ``pyaedt.application``.
```

Comment:
```suggestion
        AEDT application from the ``pyaedt.application`` class.
```
A code entity should have a noun after it to explain what type of object it is.